lib/common/watchdog.c

```diff
@@ -112,10 +112,10 @@ panic_local(void)
 
     if(ppid > 1) {
         /* child daemon */
-        exit(CRM_EX_PANIC);
+        crm_exit(CRM_EX_PANIC);
     } else {
         /* pacemakerd or orphan child */
-        exit(CRM_EX_FATAL);
+        crm_exit(CRM_EX_FATAL);
```

Comment:
I'm guessing this was intended to exit as fast as possible, since it's for a panic, but crm_exit() doesn't really do anything that would be a problem. Also this is just a fallback if sysrq *and* reboot fail, so it wouldn't be a big deal anyway.